src/Servers/IIS/AspNetCoreModuleV2/RequestHandlerLib/filewatcher.cpp

```diff
@@ -175,19 +169,8 @@ Win32 error
     LOG_INFO(L"Starting file watcher thread");
     DBG_ASSERT(pFileMonitor != nullptr);
 
-    HANDLE events[2] = { pFileMonitor->m_hCompletionPort, pFileMonitor->m_pShutdownEvent };
-
-    DWORD dwEvent = 0;
     while (true)
     {
-        // Wait for either a change notification or a shutdown event.
-        dwEvent = WaitForMultipleObjects(ARRAYSIZE(events), events, FALSE, INFINITE) - WAIT_OBJECT_0;
```

Comment:
Would another mitigation be to reduce the wait time to something less than `INFINITE`?